src/ngResource/resource.js

```diff
@@ -443,10 +443,14 @@ angular.module('ngResource', ['ng']).
           });
 
           params = params || {};
-          forEach(self.urlParams, function(_, urlParam) {
+          forEach(self.urlParams, function(paramInfo, urlParam) {
             val = params.hasOwnProperty(urlParam) ? params[urlParam] : self.defaults[urlParam];
             if (angular.isDefined(val) && val !== null) {
-              encodedVal = encodeUriSegment(val);
+              if (paramInfo.isQueryParamiValue === true) {
```

Comment:
I'm sorry for that. Actually I thought I pushed just to my `origin` and not to `upstream` as well. Still didn't manage to run the tests. Will do ASAP